src/FiniteDifferences.jl

```diff
@@ -49,6 +54,7 @@ macro     d2(A::Symbol)  esc(:( ($A[$ixi+1] - $A[$ixi])  -  ($A[$ixi] - $A[$ixi-
 macro    all(A::Symbol)  esc(:( $A[$ix  ] )) end
 macro    inn(A::Symbol)  esc(:( $A[$ixi ] )) end
 macro     av(A::Symbol)  esc(:(($A[$ix] + $A[$ix+1] )*0.5 )) end
+macro   harm(A::Symbol)  esc(:(2/(1/$A[$ix] + 1/$A[$ix+1]) )) end
```

Comment:
Agree, I see slightly better performance with your version, except when `A` is an integer.  I guess that it will be used with floats most of the time anyway.